provider/fs/src/export/serializers/mod.rs

```diff
@@ -22,6 +22,7 @@ use icu_provider::datagen::*;
 use icu_provider::prelude::*;
 use std::io;
 
+#[doc(hidden)]
```

Comment:
Issue: I don't think this should be `#[doc(hidden)]`. It is a public API that is linked to from other APIs and it is useful in terms of seeing what concrete items implement it. The feedback on this trait was:

> Should explain that this is a trait not intended to implement yourself and you should use one of the three built in serializers